sympy/sets/sets.py

```diff
@@ -1808,6 +1836,43 @@ def __lt__(self, other):
         return self.is_proper_subset(other)
 
 
+class SymmetricDifference(Set):
+    """Representes the Symmetric Difference of a set with another set.
```

Comment:
@skirpichev Could you tell me what you mean by "You could add reference here, i.e. on the wiki page."? I added the definition from the wikipedia page in docstring.